src/features/articles/components/article-editor/plugins/executable-code-block.ts

```diff
@@ -0,0 +1,154 @@
+import { addListToDropdown, Collection, Command, createDropdown, Plugin, ViewModel as Model } from "ckeditor5";
+import Node from "@ckeditor/ckeditor5-engine/src/model/node";
+
+const EXECUTABLE = "executable";
+const RUNTIMES = [
+  { value: "php", label: "PHP" },
```

Comment:
these are not valid runtimes. the valid runtimes can be found here: https://github.com/Tarhche/backend/blob/main/application/code/runCode/request.go#L20